Find a functionally equivalent code:android.animation.AnimatorSet.setStartDelay:COMMENT
Method Modifier: public      
Comment:/**
 * The amount of time, in milliseconds, to delay starting the animation after
 * {@link #start()} is called.
 *
 * @param startDelay The amount of the delay, in milliseconds
 */

@@ -1,3 +1,6 @@
 {
+    if (mStartDelay > 0) {
+        mReversible = false;
+    }
     mStartDelay = startDelay;
 }

